assets/scss/main.scss

```diff
@@ -0,0 +1,66 @@
+// Import any new stylesheets here
+
+@import "support/functions";
+@import "variables";
+@import "variables_project";
+@import "support/mixins";
+
+@import "../vendor/bootstrap/scss/bootstrap";
+
+@import "../vendor/Font-Awesome/scss/fontawesome.scss";
+@import "../vendor/Font-Awesome/scss/solid.scss";
+@import "../vendor/Font-Awesome/scss/brands.scss";
+
+@import "support/utilities";
+@import "colors";
+@import "boxes";
+@import "blog";
+@import "code";
+@import "nav";
+@import "sidebar-tree";
+@import "sidebar-toc";
+// @import "buttons";
```

Comment:
Just curious why buttons isn't being included. Aren't we using BS buttons?

Reply:
Took a look at this again - the file being left out here is a Docsy modifier for Bootstrap buttons that was giving them the "rounded" look. However, checking again I found a variable to override this. So, uncommented this and added the variable to `_variables_project` to disable it so the functionality isn't lost.